docs/xdr/features/collect/ingestion_methods/sekoiaio_docker_concentrator.md

```diff
@@ -0,0 +1,435 @@
+# SEKOIA.IO Docker concentrator
+
+## Overview
+
+[Docker](https://docs.docker.com/get-started/overview/) is a tool that can be used to run packaged applications in an isolated environment on a host.
+Packaged applications are stored in an object called an image, which includes an OS, the dependencies and the configuration. With that, the application will have the same behaviour whatever the OS used on the host as long as it's a x86-64 Linux host.
+
+SEKOIA.IO offers a preconfigured concentrator based on Docker to forward events on the platform.
+
+This method simplfies as much as possible the configuration needed to set up a concentrator in order to collect logs and send them on the each relevant Intakes.
+
+!!! Warning
+    In this method each technology MUST send their logs on different ports (of your choice) of the concentrator in order to make it work. The main principale of this methode is to discriminate each technology by port to link them with the right Intake key.

```

Comment:
In this method each technology MUST send their logs on different ports (of your choice) of the concentrator in order to make it work. The main principle of this method is to discriminate each technology by port to link them with the right Intake key.